util/gitutil/git_cli.go

```diff
@@ -276,8 +284,11 @@ func (cli *GitCLI) Run(ctx context.Context, args ...string) (_ []byte, err error
 	}
 }
 
-func getGitSSHCommand(knownHosts string, logLevel GitLogLevel) string {
+func getGitSSHCommand(knownHosts string, logLevel GitLogLevel, existingSSHCommand string) string {
 	gitSSHCommand := "ssh -F /dev/null"
+	if existingSSHCommand != "" {
+		gitSSHCommand = existingSSHCommand
```

Comment:
this will help us maintain this for in the future if things change upstream:
```suggestion
		gitSSHCommand = existingSSHCommand // earthly-specific
```

Reply:
Ah, good catch.